lib/matplotlib/axes/_axes.py

```diff
@@ -7390,14 +7390,23 @@ def spy(self, Z, precision=0, marker=None, markersize=None,
 
     def matshow(self, Z, **kwargs):
         """
-        Plot a matrix or array as an image.
+        Plot the values of a 2D matrix or array as color-coded image.
+
+        This is just a convenience function wrapping `.imshow` to set useful
```

Comment:
I think it might be worth putting this (rather technical list) below in a notes section, since the average user probably just wants to see a matrix and not bother about imshow options.